<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Show background</string>
      </property>
     </widget>
    </item>
    <!--
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Popup Timeout</string>
      </property>
     </widget>
 </item> -->
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Popup Mode</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="popupMode">
        <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum> 
        </property>
        <item>
            <property name="text">
                <string>Close when loosing focus</string>
            </property>
         </item>
         <item>
            <property name="text">
                <string>Close manually</string>
            </property>
                   </item>
       <!--  <item>
             <property name="text">
                <string>Close after timeout</string>
            </property>
         </item>
         -->

     </widget>
    </item>
    <!-- <item row="1" column="1">
     <widget class="QSpinBox" name="popdownTimeout"/>
 </item> -->
    <item row="2" column="1">       
     <widget class="QComboBox" name="showBackground">
        <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum> 
        </property>
        <item>
            <property name="text">
                <string>No background</string>
            </property>
         </item>
         <item>
            <property name="text">
                <string>Translucent background</string>
            </property>
         </item>
         <item>
             <property name="text">
                <string>Standard background</string>
            </property>
         </item>
     </widget>
     
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
